test-framework/maven/src/main/java/io/quarkus/maven/it/verifier/MavenProcessInvoker.java

```diff
@@ -56,6 +56,9 @@ public InvocationResult execute(InvocationRequest request) throws MavenInvocatio
         Commandline cli;
         try {
             cli = cliBuilder.build(request);
+            if (logger != null) {
+                logger.info("Running Maven CLI command: " + cli);
```

Comment:
Do we really want this to be INFO?

Reply:
If CI shows an error from one of these tests, you would right away have the info on what was executed. As opposed to making it DEBUG, and then having to run this with debug locally to see what got executed. I'm fine either way TBH. @zakkak wanted to see this logged, WDYT?

Reply:
I would prefer it to stay at INFO level, but DEBUT would be OK as well.

Reply:
I've switched it to DEBUG.